addons/survey/static/src/question_page/description_page_field.js

```diff
@@ -0,0 +1,27 @@
+/** @odoo-module */
+
+import { CharField } from "@web/views/fields/char/char_field";
+import { registry } from "@web/core/registry";
+
+const { useEffect, useRef } = owl;
+
+class DescriptionPageField extends CharField {
```

Comment:
Hello @Polymorphe57 

May I ask why you did not remove the file implementing this in the legacy code in this PR?
I was greping "web.basic_fields" today to see if we missed converting things and thought this was not ported, when it actually was ^^

I'm talking about this file:
https://github.com/odoo/odoo/blob/16.0/addons/survey/static/src/js/fields_form_page_description.js

Reply:
Hello @aab-odoo 

Just to make sure I understand, there is still in v16.0 a way to open the legacy form view of the survey.survey model and get this legacy widget?

Or is that a general rule that legacy widgets should not be removed?

Because I'm pretty sure internally in SM team we removed the legacy code when porting to OWL, so if it can break things maybe we need to double-check :grimacing: .

Thanks for the quick answer!

Reply:
Hello @awa-odoo 

Yes, it is still possible. All kanban and list views can still be rendered with the legacy implementation, you just need to enter studio. For form views, it's also possible but less likely, as the only way (I think) is through a legacy FormViewDialog. That dialog is only used in legacy code (list/kanban/fields). Since in Studio, views are inert, the list/kanban source should not be an issue (you won't be able to open a FormViewDialog from there). But for fields it might be (for instance, from a legacy Many2one field used in a legacy ClientAction).

So **theoretically**, legacy fields should not have been removed. In web, we kept all legacy fields. I saw that approximately half of the legacy fields have been removed in the other addons, but I saw that too late. We probably didn't communicate enough, our fault. Hopefully, it could be fine, for several reasons:
 1) if the field (e.g. "partner_many2one") can't be found in the registry, we simply fallback on the field type (e.g. "many2one"), and as I said, all those fields have been kept. In Studio, this won't be a problem for most of the cases (especially because it's only for kanban and list) as you won't even see the difference.
 2) in the few legacy FormViewDialogs that can still be accessed, the chance that one of the removed legacy fields is used is very low. And even if we found cases, we can still add the legacy code back, case by case.